phongo_version.h

```diff
@@ -22,7 +22,7 @@
  * publishing a release. */
 
 /* clang-format off */
-#define PHP_MONGODB_VERSION "1.18.2-dev"
+#define PHP_MONGODB_VERSION "1.18.2dev"
```

Comment:
During testing, I was trying to create a package with `devel` stability and PECL complained that the `-dev` suffix is wrong. I dislike that difference, but hopefully it won't be much longer until we can finally say goodbye to pecl.